06_PDDE_Checker.R

```diff
@@ -84,6 +89,7 @@ missing_CoC_Info <- Project0() %>%
   )
 
 missing_CoC_Geography <- missing_CoC_Info %>%
+  filter(VictimServiceProvider!=1 & HousingType!=3) %>%
```

Comment:
Small thing, but I think it would be best to replace the 3 here with a new tenant_scattered_site variable in the hardcodes.R script so it's more clear what's being filtered (out) here

Reply:
Done